keps/sig-node/2238-liveness-probe-grace-period/kep.yaml

```diff
@@ -5,7 +5,7 @@ authors:
 owning-sig: sig-node
 participating-sigs:
   - sig-node
-status: implementable
+status: implemented
```

Comment:
Assuming this is correct since it went in in v.121

Reply:
Per https://github.com/kubernetes/enhancements/issues/2238#issuecomment-1159882321 it was not, so will revert